<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulation_ResultClass</class>
 <widget class="QMainWindow" name="Simulation_ResultClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>仿真结果</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OK">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="result_test">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>256</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>路口类型：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>配时方案：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="road_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="signal_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
